app/models/repo_solr_client.rb

```diff
@@ -25,6 +32,15 @@ def get_doc(uuid)
       resp['response']
     end
   end
+  
+  def get_number_of_children_for_parent_uuid(uuid)
+    if @repo_solr_client
+      # Find out how many actual items are beneath this parent.
+      @solr_params = { q: "parentUUID:\"#{uuid}\" AND type_s:Item" }
+      resp = @repo_solr_client.get 'select', params: @solr_params
+      resp['response']['numFound']
```

Comment:
we do, but I think that's a good thing. any errors would be failed delayed jobs to look into.

Reply:
logically speaking, we cannot proceed if we can't get the parent information, so abort.

Reply:
Understood. Is it possible to bubble something up about why the job failed in that case? 

For example, when ingest jobs are failing, someone would need to look into the db, find a failed job, find the last error (no method [] for nil, or similar), find the last line in the stack trace that's one of our lines of code, look into the code base, and then understand that it's probably a connectivity issue with solr that's at fault rather than bad data or a bug.

Could we check to see that resp['response'] exists, and if not, raise 'bad response from solr'?

Reply:
Added code to raise an error here and in get_doc method.